src/Microsoft.AspNet.Authentication/DataHandler/TicketDataFormat.cs

```diff
@@ -7,7 +7,8 @@ namespace Microsoft.AspNet.Authentication
 {
     public class TicketDataFormat : SecureDataFormat<AuthenticationTicket>
     {
-        public TicketDataFormat(IDataProtector protector) : base(new TicketSerializer(), protector)
+        public TicketDataFormat(IDataProtector protector)
+            : base(TicketSerializer.Default, protector)
```

Comment:
I updated this constructor to use the static instance (that was not used at all in the current code base :weary:)